.github/workflows/deploy-react-app.yaml

```diff
@@ -36,6 +45,28 @@ on:
 
 jobs:
 
+  derive-commit-hash:
```

Comment:
See comment above re opportunity to refactor

Reply:
ticket to be created

Reply:
DR-1028 to cover refactoring